### PR TITLE
'impor' -> 'import'

### PR DESCRIPTION
**Pull request by Purvanshsingh**

### CODE REVIEW

**Review by rtgdk — DISMISSED** [review]

**Comment by rtgdk** [comment]
@Purvanshsingh Can you please signoff the commit? DCO check is failing due to that.

File: spdx/document.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+f
```
Comment (carmenbianca) [inline spdx/document.py]: This is probably unintended.

**Comment by lhh** [comment]
Looks like this PR is obsolete at this point.

**Review by nicoweidner — COMMENTED** [review]
This is already fixed on `main`, PR can be closed

**Pull request closed without merging**